src/content/en/updates/2018/09/asynchronous-access-to-http-cookies.md

```diff
@@ -0,0 +1,187 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: The Cookie Store API offers asynchronous acesss to HTTP cookies, and opens up the cookie jar to service workers.
+
+{# wf_updated_on: 2018-09-01 #}
+{# wf_published_on: 2018-07-23 #}
+{# wf_tags: cookie,chrome69 #}
+{# wf_featured_image: /web/updates/images/generic/styles.png #}
+{# wf_featured_snippet: The Cookie Store API offers asynchronous acesss to HTTP cookies, and opens up the cookie jar to service workers. #}
+{# wf_blink_components: Blink>WebVR #}
+
+# Asynchronous Access to HTTP Cookies {: .page-title }
+
+{% include "web/_shared/contributors/victorcostan.html" %}
+
+The [Cookie Store API](https://www.chromestatus.com/feature/5658847691669504) is
+available for Origin Trials starting in Chrome 69. The API introduces the
+following exciting possibilities.
+
+* Cookies can be accessed asynchronously, avoiding jank on the main thread.
+* Changes to cookies can be observed, avoiding polling.
+* Cookies can be accessed from service workers.
+
+## You (probably) don't need cookies
+
+Before diving into the new API, I'd like to state that cookies are still the Web
+platform's worst client-side storage primitive, and should still be used as a
+last resort. This isn't an accident - cookies are the Web's first client-side
```

Comment:
"cookies `were` the Web's first client-side"

Reply:
Done.